buildpack/telemetry/metrics.py

```diff
@@ -612,7 +669,9 @@ def _inject_user_session_metrics(self, stats):
 
     @property
     def _select_stats_to_emit(self):
-        return [self._inject_user_session_metrics]
+        if not self.micrometer_metrics_enabled:
+            return [self._inject_user_session_metrics]
```

Comment:
dont understand this. No user sesison metrics when micrometer is enabled?

Reply:
We don't need user session metrics from the old stream. We'll push it from micrometer via telegraf.